x-pack/plugins/endpoint/public/applications/endpoint/index.tsx

```diff
@@ -8,24 +8,43 @@ import * as React from 'react';
 import ReactDOM from 'react-dom';
 import { CoreStart, AppMountParameters } from 'kibana/public';
 import { I18nProvider, FormattedMessage } from '@kbn/i18n/react';
+import { History, createHashHistory } from 'history';
+import { Route, Router, Switch } from 'react-router-dom';
 
 /**
  * This module will be loaded asynchronously to reduce the bundle size of your plugin's main bundle.
  */
 export function renderApp(coreStart: CoreStart, { element }: AppMountParameters) {
   coreStart.http.get('/api/endpoint/hello-world');
 
-  ReactDOM.render(<AppRoot />, element);
+  const history = createHashHistory()
```

Comment:
Kibana testing utils all seem to use hash history (see test below), doesn't mean we have to